pom.xml

```diff
@@ -32,7 +32,7 @@
             <dependency>
                 <groupId>net.openhft</groupId>
                 <artifactId>chronicle-bom</artifactId>
-                <version>2.17.525</version>
+                <version>2.17.600</version>
```

Comment:
latest is 2.19.34, you might wanna use that instead

Reply:
I saw that but I didn't know if that was safe or not. I've updated both BOMs to the lastest available.